lib/postinst-migrations.sh

```diff
@@ -71,4 +71,18 @@ except Exception as e:
     print('Error while creating public user-config file: {}'.format(e))
 "
     fi
+    # Create an only series marker file if not present
+    # This is to ensure that the only-series contract check runs correctly
+    if [ ! -f "/var/lib/ubuntu-advantage/private/machine-token.json" ]; then
+        /usr/bin/python3 -c "
+from uaclient.files import machine_token
+machine_token_file = machine_token.get_machine_token_file()
+only_series = machine_token_file.only_series()
+    if only_series:
+        print(only_series)
+"
```

Comment:
Let's wrap this on `try..except` just to be extra safe, as we don't want the postinst to fail if somehow that code fails

Reply:
Updated @lucasmoura